internal/fuse/file.go

```diff
@@ -154,13 +155,6 @@ func (f *file) Read(ctx context.Context, req *fuse.ReadRequest, resp *fuse.ReadR
 	return nil
 }
 
-func (f *file) Release(ctx context.Context, req *fuse.ReleaseRequest) error {
```

Comment:
Shouldn't this clear the `cached` field? Is it that the fuse library drops the `file` struct anyways afterwards such that this isn't necessary?

Reply:
The struct is dropped. Release is for closing files and network connections.